[EDU-626] Add ECR deprecation notice (#513)
seqeralabs/docs: changelog/seqera-enterprise/v24.2.mdx

## Breaking changes and warnings

### Seqera AWS ECR repository customer access ends June 1, 2025

> Customers will no longer be able to pull Seqera Enterprise container images from the legacy Seqera AWS ECR repository after June 1, 2025. All Seqera Enterprise images must be retrieved via the `cr.seqera.io` container registry after this cutoff date. The installation and configuration templates provided for both [Docker Compose](https://docs.seqera.io/platform/24.2/enterprise/docker-compose) and [Kubernetes](https://docs.seqera.io/platform/24.2/enterprise/kubernetes) installations already reference the `cr.seqera.io` container image URLs. If you have not yet transitioned to this registry, contact support to request credentials and for any further assistance. 

Reviewer: ```suggestion
Customers will no longer be able to pull Seqera Enterprise container images from the legacy Seqera AWS ECR repository after June 1, 2025. All Seqera Enterprise images must be retrieved via the `cr.seqera.io` container registry after this cutoff date. The installation and configuration templates provided for both [Docker Compose](https://docs.seqera.io/platform/24.2/enterprise/docker-compose) and [Kubernetes](https://docs.seqera.io/platform/24.2/enterprise/kubernetes) installations already reference the `cr.seqera.io` container image URLs. If you have not yet transitioned to this registry, contact Support to request credentials and for any further assistance. 
```

Reviewer: Should we add a Support link?

> Customers will no longer be able to pull Seqera Enterprise container images from the Seqera AWS ECR repository after June 1, 2025. All Seqera Enterprise images must be retrieved via Harbor at cr.seqera.io after this cutoff date. The installation and configuration templates provided for both [Docker Compose](https://docs.seqera.io/platform/24.2/enterprise/docker-compose) and [Kubernetes](https://docs.seqera.io/platform/24.2/enterprise/kubernetes) installations already reference the cr.seqera.io container image URLs to retrieve your containers via Harbor. If you have not yet transitioned to our Harbor registry, contact support to request credentials and for any further assistance. 

Reviewer 2: ```suggestion
Customers will no longer be able to pull Seqera Enterprise container images from the legacy Seqera AWS ECR repository after June 1, 2025. All Seqera Enterprise images must be retrieved via the `cr.seqera.io` Container Registry after this cutoff date. The installation and configuration templates provided for both [Docker Compose](https://docs.seqera.io/platform/24.2/enterprise/docker-compose) and [Kubernetes](https://docs.seqera.io/platform/24.2/enterprise/kubernetes) installations already reference the `cr.seqera.io` Container Registry. If you have not yet transitioned to our Harbor registry, contact support to request credentials and for any further assistance. 
```

Reviewer 2: I'd remove the mention to Harbor, not many people know what that is and they probably don't need to know it at all, given that we don't mention "Harbor" anywhere else in the docs.
